META6.json

```diff
@@ -9,7 +9,7 @@
     "provides" : {
         "P6doc" : "lib/P6doc.pm6"
     },
-    "depends" : ["JSON::Fast", "File::Find"],
+    "depends" : ["JSON::Fast:ver<0.9.18>", "Path::Finder:ver<0.2.0>", "Perl6::Documentable:ver<1.1.2>", "Pod::Load:ver<0.5.2>"],
```

Comment:
Generally I'd pin to a *minimum* version, allowing higher versions to be used (the `+` in the code below). Generally I'd also prefer to stick to a single `:api`, but none of the modules used expose this in their `META6.json`.

    depends: [
      "JSON::Fast:ver<0.9.18+>",
      "Path::Finder:ver<0.2.0+>",
      "Perl6::Documentable:ver<1.1.2+>",
      "Pod::Load:ver<0.5.2+>"
    ],

Reply:
The only dependency where I actually have to pin a specific version for now is `Perl6::Documentable`, since the exact version is the required.